README.md

```diff
@@ -59,6 +59,28 @@ from other services too: you'll just need to provide your file URLs for import.
                                   check requests.
                                   
   ```              
+ 
+## Output file format
+The output file format is quite simple.
+For each input file URL of Filestack file handle, there are 3 columns divided by the \t symbol (tab).
+
+The first column holds input file URL or Filestack handle itself, 
+second column — upload operation status: success or failed, 
+third column — output Uploadcare URL or error description.
+
+For instance, you're willing to migrate the three following files to Uploadcare 
+where the first two are presented as URL and the third one as Filestack file 
```

Comment:
URL can normally be used in its plural form as `URLs`, e.g. "Overuse of hyphens in URLs can be seen as spammy, so it’s best to use caution and limit hyphen use in URLs when possible"